The main People list is returning 500 server error

This is almost certainly related to recent timezone fixes.

```
[2019-01-15 05:10:09 +0000] [9] [ERROR] Error handling request /people/16565393
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/dist-packages/gunicorn/workers/sync.py", line 135, in handle
    self.handle_request(listener, req, client, addr)
  File "/usr/local/lib/python3.6/dist-packages/gunicorn/workers/sync.py", line 176, in handle_request
    respiter = self.wsgi(environ, resp.start_response)
  File "/usr/local/lib/python3.6/dist-packages/flask/app.py", line 1997, in __call__
    return self.wsgi_app(environ, start_response)
  File "/usr/local/lib/python3.6/dist-packages/flask/app.py", line 1985, in wsgi_app
    response = self.handle_exception(e)
  File "/usr/local/lib/python3.6/dist-packages/flask/app.py", line 1540, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib/python3.6/dist-packages/flask/_compat.py", line 33, in reraise
    raise value
  File "/usr/local/lib/python3.6/dist-packages/flask/app.py", line 1982, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.6/dist-packages/flask/app.py", line 1614, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.6/dist-packages/flask/app.py", line 1517, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib/python3.6/dist-packages/flask/_compat.py", line 33, in reraise
    raise value
  File "/usr/local/lib/python3.6/dist-packages/flask/app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python3.6/dist-packages/flask/app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/usr/local/lib/python3.6/dist-packages/freezing/web/views/people.py", line 83, in people_show_person
    if week_start <= r.start_date <= week_end:
TypeError: can't compare offset-naive and offset-aware datetimes
```

--- a/freezing/web/config.py
+++ b/freezing/web/config.py
@@ -17,6 +17,7 @@ if os.path.exists(envfile):
 
 _basedir = os.path.abspath(os.path.join(os.path.dirname(__file__), '..', '..'))
 
+
 class Config:
 
     DEBUG: bool = env('DEBUG', cast=bool, default=False)

--- a/freezing/web/views/people.py
+++ b/freezing/web/views/people.py
@@ -47,7 +47,8 @@ def people_list_users():
         for r in u.rides:
             total_rides += 1
             total_dist += r.distance
-            if week_start <= r.start_date.replace(tzinfo=timezone(r.timezone)).date() <= week_end:
+            ride_date = r.start_date.replace(tzinfo=timezone(r.timezone)).date()
+            if week_start <= ride_date <= week_end:
                 weekly_dist += r.distance
                 weekly_rides += 1
         users.append({"name": u.display_name,
@@ -80,7 +81,8 @@ def people_show_person(user_id):
     for r in our_user.rides:
         total_rides += 1
         total_dist += r.distance
-        if week_start <= r.start_date.replace(tzinfo=timezone(r.timezone)).date() <= week_end:
+        ride_date = r.start_date.replace(tzinfo=timezone(r.timezone)).date()
+        if week_start <= ride_date <= week_end:
             weekly_dist += r.distance
             weekly_rides += 1
     return render_template('people/show.html', data={

--- a/freezing/web/views/people.py
+++ b/freezing/web/views/people.py
@@ -9,7 +9,7 @@ from freezing.model.orm import Team, Athlete
 
 from freezing.web import config
 
-from pytz import utc
+from pytz import utc, timezone
 
 
 blueprint = Blueprint('people', __name__)
@@ -35,8 +35,8 @@ def get_today():
 @blueprint.route("/")
 def people_list_users():
     users_list = meta.scoped_session().query(Athlete).filter(Athlete.team.has(leaderboard_exclude=0)).order_by(Athlete.name)  # @UndefinedVariable
-    tdy = get_today()
-    week_start = tdy - timedelta(days=(tdy.weekday()) % 7)
+    today = get_today()
+    week_start = today.date() - timedelta(days=(today.weekday()) % 7)
     week_end = week_start + timedelta(days=6)
     users = []
     for u in users_list:
@@ -47,7 +47,7 @@ def people_list_users():
         for r in u.rides:
             total_rides += 1
             total_dist += r.distance
-            if week_start <= r.start_date <= week_end:
+            if week_start <= r.start_date.replace(tzinfo=timezone(r.timezone)).date() <= week_end:
                 weekly_dist += r.distance
                 weekly_rides += 1
         users.append({"name": u.display_name,
@@ -70,8 +70,8 @@ def people_show_person(user_id):
         abort(404)
 
     our_team = meta.scoped_session().query(Team).filter_by(id=our_user.team_id).first()
-    tdy = get_today()
-    week_start = tdy - timedelta(days=(tdy.weekday()) % 7)
+    today = get_today()
+    week_start = today.date() - timedelta(days=(today.weekday()) % 7)
     week_end = week_start + timedelta(days=6)
     weekly_dist = 0
     weekly_rides = 0
@@ -80,7 +80,7 @@ def people_show_person(user_id):
     for r in our_user.rides:
         total_rides += 1
         total_dist += r.distance
-        if week_start <= r.start_date <= week_end:
+        if week_start <= r.start_date.replace(tzinfo=timezone(r.timezone)).date() <= week_end:
             weekly_dist += r.distance
             weekly_rides += 1
     return render_template('people/show.html', data={

--- a/setup.py
+++ b/setup.py
@@ -15,7 +15,7 @@ from setuptools import setup, find_packages
 __authors__ = ['"Hans Lellelid" <hans@xmpl.org>']
 __copyright__ = "Copyright 2018 Hans Lellelid"
 
-version = '1.1.8'
+version = '1.1.9'
 
 long_description="""
 The freezing saddles cycling competition website/scoreboard.

